app/models/subject.js

```diff
@@ -17,7 +17,7 @@ export default Fragment.extend(Validations, {
   subject: attr('string'),
   subjectScheme: attr('string', { defaultValue: null }),
   schemeUri: attr('string', { defaultValue: null }),
-  valueUri: attr('string', { defaultValue: "" }),
+  valueUri: attr('string', { defaultValue: null }),
```

Comment:
I set this to `null` because it is always empty unless specified in xml file. so since it is an optional parameter we can remove it if its null, unless it was by design to keep it like that.
<img width="876" alt="Screenshot 2023-10-06 at 11 33 40" src="https://github.com/datacite/bracco/assets/9620622/8e5aec6e-c87d-441d-9831-910f461c81c5">

Reply:
If we keep value as `null` then it does not appear in XML?

Reply:
It was `null` before, but then we changed it to `""` in this PR https://github.com/datacite/bracco/commit/f8cf3edbf9bf171089baa25febcad2440206b3cd

Reply:
interesting, I am sure there was some logic behind. @jrhoads

Reply:
At some point, there was an error when trying to save these to lupo if there was a null value.  That may be resolved.